plans/ostrava.yml

```diff
@@ -177,81 +180,81 @@
   - name: Poznámka o importování
     type: lesson
-    link: http://naucse.python.cz/2017/pyladies-ostrava-podzim/beginners/circular-imports/
+    link: http://naucse.python.cz/2018/pyladies-ostrava-jaro/beginners/circular-imports/
   - name: Domácí projekty (PDF)
     type: homework
     link: v1/s005-modules/handout/handout5b-ostrava.pdf
   - name: Odevzdání domácích projektů
     type: link
-    link: https://www.dropbox.com/request/mlgOENzebT3oRXEBdvMt
-  - name: Feedback k domácím projektům
-    type: link
-    link: v1/s005-modules/ostrava/feedback-homeworks-modules.html
+    link: https://www.dropbox.com/request/GtE0mpyMT4nLN2Lt8NGV
+  # - name: Feedback k domácím projektům
+  #   type: link
+  #   link: v1/s005-modules/ostrava/feedback-homeworks-modules.html
 - name: Testování a procvičování
-  date: 2017-11-14
+  date: 2018-04-11
   materials:
   - name: Testování
     type: lesson
-    link: http://naucse.python.cz/2017/pyladies-ostrava-podzim/beginners/testing/
+    link: http://naucse.python.cz/2018/pyladies-ostrava-jaro/beginners/testing/
   - name: Domácí projekty (PDF)
     type: homework
     link: v1/s005-modules/handout/handout5c-ostrava.pdf
   - name: Odevzdání domácích projektů
     type: link
-    link: https://www.dropbox.com/request/3HlBHVc8gAvZyCYZ9LvP
+    link: https://www.dropbox.com/request/grRZe5fq8BtJQvJzZtcd
 - name: Grafika
-  date: 2017-11-21
+  date: 2018-04-18
   materials:
   - name: Grafika
     type: lesson
-    link: http://naucse.python.cz/2017/pyladies-ostrava-podzim/intro/pyglet/
+    link: http://naucse.python.cz/2018/pyladies-ostrava-jaro/intro/pyglet/
   - name: "Praktické cvičení: Pong - mimo kurz navíc"
     type: lesson
-    link: http://naucse.python.cz/2017/pyladies-ostrava-podzim/projects/pong/
+    link: http://naucse.python.cz/2018/pyladies-ostrava-jaro/projects/pong/
   - name: Kód celé hry Pong
     type: link
     link: http://pyladies.cz/v1/s012-pyglet/pong.py
   - name: Tahák na Pyglet
     type: handout
     link: https://github.com/pyvec/cheatsheets/raw/master/pyglet/pyglet-basics-cs.pdf
 - name: Třídy
-  date: 2017-11-28
+  date: 2018-04-25
   materials:
   - name: Třídy
     type: lesson
-    link: http://naucse.python.cz/2017/pyladies-ostrava-podzim/beginners/class/
+    link: http://naucse.python.cz/2018/pyladies-ostrava-jaro/beginners/class/
   - name: Dědičnost
     type: lesson
-    link: http://naucse.python.cz/2017/pyladies-ostrava-podzim/beginners/inheritance/
+    link: http://naucse.python.cz/2018/pyladies-ostrava-jaro/beginners/inheritance/
 - name: Závěrečný projekt
-  date: 2017-12-05
+  date: 2018-05-02
   materials:
   - name: Asteroids
     type: lesson
-    link: http://naucse.python.cz/2017/pyladies-ostrava-podzim/projects/asteroids/
+    link: http://naucse.python.cz/2018/pyladies-ostrava-jaro/projects/asteroids/
   - name: Množinový tahák
     type: handout
     link: https://github.com/pyvec/cheatsheets/blob/master/sets/sets-cs.pdf
```

Comment:
Dal bych spíše _raw_ než _blob_, ať je to jednotné s ostatními taháky. https://github.com/pyvec/cheatsheets/raw/master/sets/sets-cs.pdf